src/main/java/uk/gov/ons/ctp/response/notify/service/impl/NotifyServiceImpl.java

```diff
@@ -149,8 +149,13 @@ private ActionFeedback processSms(ActionRequest actionRequest) throws Notificati
             + " for actionId = {}", templateId, phoneNumber, personalisation, actionId);
 
         SendSmsResponse response = notificationClient.sendSms(templateId, phoneNumber, personalisation, null);
-        log.debug("status = {} for actionId = {}", notificationClient.getNotificationById(
-            response.getNotificationId().toString()).getStatus(), actionId);
+
+        if (response != null) {
+            log.debug("status = {} for actionId = {}", notificationClient.getNotificationById(
+                    response.getNotificationId().toString()).getStatus(), actionId);
+        } else {
+            log.debug("response is null");
```

Comment:
Probably still worth logging the action id here as that's not null

Reply:
As you've done below

Reply:
Ummm it's always logged several lines above but I've added it again as requested

Reply:
Missed that tbh, not sure it's worth logging it twice?